tests/integration-tests/configs/common/common.yaml

```diff
@@ -56,7 +56,7 @@ cloudwatch_logging:
       # 2) run the test for all OSes with slurm
       - regions: ["ap-east-1"]
         instances: {{ common.INSTANCES_DEFAULT_ARM }}
-        oss: ["ubuntu2204", "rhel8"]
+        oss: ["ubuntu2204", "rocky8"]
```

Comment:
There is a problem with this change.

We cannot add rocky8 in common, because this test will also be used to test released version and we're not sure we'll release official pcluster rocky8 images, so these tests will fail.

If you need to enable these tests, you need to "override" them in develop.yaml

Reply:
Sure